components/atoms/Card.js

```diff
@@ -9,13 +8,7 @@ export default function Card(props) {
   return (
     <div className="text-black text-left card rounded-tl-md rounded-b-md border my-6 mx-4">
       <div>
-        <Image
-          src={props.image}
-          alt={props.imgalt}
-          width={252}
-          height={101}
-          layout="intrinsic"
-        ></Image>
+        <img src={props.image} width={252} height={101} layout="intrinsic" />
```

Comment:
```suggestion
        <img src={props.image} width={252} height={101} layout="intrinsic" alt={props.imgalt} />
```
Maybe adding this back will fix the issue? Then a null image alt will get passed in and axe will recognize it as a decorative image

Reply:
added alt=""

Reply:
Looks good to me now. LGTM